dev/a11y_assessments/lib/use_cases/navigation_bar.dart

```diff
@@ -0,0 +1,79 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+import 'use_cases.dart';
+
+class NavigationBarUseCase extends UseCase {
+
+  @override
+  String get name => 'NavigationBar';
+
+  @override
+  String get route => '/navigation-bar';
+
+  @override
+  Widget build(BuildContext context) => const MainWidget();
+}
+
+class MainWidget extends StatefulWidget {
+  const MainWidget({super.key});
+
+  @override
+  State<MainWidget> createState() => MainWidgetState();
+}
+
+class MainWidgetState extends State<MainWidget> {
+  int currentPageIndex = 0;
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        backgroundColor: Theme.of(context).colorScheme.inversePrimary,
+        title: const Text('NavigationBar'),
+      ),
+      bottomNavigationBar: NavigationBar(
+        onDestinationSelected: (int index) {
+          setState(() {
+            currentPageIndex = index;
+          });
+        },
+        indicatorColor: Colors.amber[800],
+        selectedIndex: currentPageIndex,
+        destinations: const <Widget>[
```

Comment:
same here, there is no control to autofocus on

Reply:
Are any of those widgets focusable? Is the issue that they lack the `autofocus` property?

Reply:
badge by it self doesn't have a gesture recognizer. Even if it has a gesture detector, we still need some widget to possess focusnode.